test/lib/usd/utils/CMakeLists.txt

```diff
@@ -93,8 +87,7 @@ mayaUsd_add_test(${TARGET_NAME}
 # -----------------------------------------------------------------------------
 
 set(USDUTILS_PYTHON_TEST_NAME Python:MayaTransformAPI)
-add_test(
-  NAME ${USDUTILS_PYTHON_TEST_NAME}
+mayaUsd_add_test(${USDUTILS_PYTHON_TEST_NAME}
   COMMAND
     python -m unittest discover -s ${CMAKE_CURRENT_SOURCE_DIR}
 )
```

Comment:
@robthebloke  This code is not portable:

```
set_tests_properties(${USDUTILS_PYTHON_TEST_NAME}
  PROPERTIES
    ENVIRONMENT PYTHONPATH=${CMAKE_BINARY_DIR}/${INSTALL_DIR_SUFFIX}/usdutils:${CMAKE_BINARY_DIR}/${INSTALL_DIR_SUFFIX}/lib/usd/utils:$ENV{PYTHONPATH} 
)
```

Replace above with this:

```
set(python_path 
    ${CMAKE_BINARY_DIR}/${INSTALL_DIR_SUFFIX}/usdutils
    ${CMAKE_BINARY_DIR}/${INSTALL_DIR_SUFFIX}/lib/usd/utils
    $ENV{PYTHONPATH} 
)
separate_argument_list(python_path)

set_tests_properties(${USDUTILS_PYTHON_TEST_NAME}
  PROPERTIES
    ENVIRONMENT PYTHONPATH=${python_path}
)
```

Hmm, your python paths don't seem correct to me. Does this code even work?